analyzer/src/main/kotlin/managers/utils/NuGetSupport.kt

```diff
@@ -264,7 +258,31 @@ class NuGetSupport(serviceIndexUrls: List<String> = listOf(DEFAULT_SERVICE_INDEX
 /**
  * Parse information about the authors of a package from the given [spec].
  */
-private fun parseAuthors(spec: PackageSpec) = sortedSetOf(spec.metadata.authors)
+private fun parseAuthors(spec: PackageSpec?): SortedSet<String> =
+    spec?.metadata?.authors?.split(',', ';')
+        ?.map(String::trim)
+        ?.filterNot(String::isEmpty)
+        ?.toSortedSet() ?: sortedSetOf()
+
+/**
+ * Parse information about the licenses of a package from the given [spec].
+ */
+private fun parseLicenses(spec: PackageSpec?): SortedSet<String> {
```

Comment:
Nit: The (sensible) extraction of license parsing to a function could have been done in a preparing commit, as it's unrelated to the strict topic of author parsing, and it would have helper to keep the diff for this topic smaller.

Reply:
Created a new commit for this change.